### DISCUSSION TITLE
Capire da quale echo dot arriva la richiesta di attivazione di uno script

### DISCUSSION THREAD

**Discussion by akela-it** [indomus/forum#952] [HUB: Home Assistant]
Salve a tutti, 
Ho 3 echo dot e sto facendo un po' di script per farmi dare delle informazioni sullo stato dei sensori (ad esempio Alexa che temperatura c'è in camera 1 o Alexa qual'è la percentuale di umidità della stanza 2). Mi piacerebbe però mandare la risposta sull'echo dal quale sto effettuando la richiesta... c'è modo di far sapere a home assistant qual'è il dispositivo che ha attivato lo script?

Saluti

**Comment by xyzmauro** [comment]
Ciao, trovi tutto nel wiki del custom_component…
https://github.com/custom-components/alexa_media_player/wiki#creating-sensorlast_alexa

**Reply by akela-it** [reply]
OTTIMO! Grazie!
